GeoLib/GEOObjects.cpp

```diff
@@ -551,9 +551,83 @@ void GEOObjects::renameGeometry(std::string const& old_name,
     }
 }
 
-const GeoLib::GeoObject* GEOObjects::getGeoObject(const std::string &geo_name,
-                                                            GeoLib::GEOTYPE type,
-                                                            const std::string &geo_obj_name) const
+void GEOObjects::markPointsAsUsed(std::string const& geo_name,
+                                  std::vector<bool>& flags)
```

Comment:
better to return the flags instead of modifying.

Reply:
if the function does not modify anything, mark it as const.

Reply:
Minor: Maybe the name of the method could be changed to 'markUsedPoints' that is a tiny bit more readable IMHO.

Reply:
Renamed method and set const.

I can return the flags-array instead of returning it, but wouldn't that mean I'd create a copy of it? I need the vector in the calling method anyway, so if I return it here, I'd have the vector twice?!